port/zos390/omrportpg.h

```diff
@@ -48,6 +48,17 @@
 #define TTOKEN_BUF_SZ                   16
 #endif /* defined(OMR_ENV_DATA64) */
 
+typedef struct STFLEFacilities {
+	uint64_t dw1;
+	uint64_t dw2;
+	uint64_t dw3;
+} STFLEFacilities;
+
+typedef struct OMRSTFLECache {
+	uintptr_t lastDoubleWord;
+	STFLEFacilities facilities;
+} OMRSTFLECache;
+
```

Comment:
Should these structs be under an OMR_ENV_DATA64 ifdef?

Reply:
There was no ifdef in the openJ9 equivalent, so I assumed it wasn't necessary... Should I add one?

Reply:
It seems to me that these structures are only used in 64bit ZOS, so I am thinking yes.